### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zamith**

### CODE REVIEW

File: dayone/README.md [hunk 1]
```diff
@@ -1,22 +1,31 @@
 Day One Checklist
 =================
 
-This chapter contains a checklist to be done on the first day. The apprentice must learn the rules of apprenticeship and setup is work environment. If there is time left the first mentor will pick up.
+This document contains a checklist of things to be completed on the first day, and some ground rules for the apprenticeship. The apprentice must learn this rules. If there is time left the first mentor will pick up.
```
Comment (zamith) [inline dayone/README.md:4]: @gabrielpoca I did not understand the "If there is time left the first mentor will pick up." sentence.

Comment (gabrielpoca) [inline dayone/README.md:4]: It means that if the setup doesn't require all day the first mentor can start.

**Comment by gabrielpoca** [comment]
:+1:

File: dayone/README.md (hunk 1 above)
Comment (zamith) [inline dayone/README.md:4]: Oh. I propose the setup to be done on day 0, since most (all?) of it can be done asynchronously.

**Pull request merged**